servers.json

```diff
@@ -1,4 +1,12 @@
 [
+   {
+        "id": "visionmc",
+        "name": "VisionMC Network",
+        "addresses":[
+            "visionmc.club"
+            "icantconnectlmao.visionmc.gq"
```

Comment:
You do not need to specify sub-domains, Lunar Client services automatically detect them.